@here/harp-omv-datasource/lib/OmvDecodedTileEmitter.ts

```diff
@@ -422,17 +514,40 @@ export class OmvDecodedTileEmitter implements IOmvEmitter {
 
                 const lineType = isLineTechnique(technique) ? LineType.Simple : LineType.Complex;
 
-                this.applyLineTechnique(
-                    lineGeometry,
-                    technique,
-                    techniqueIndex,
-                    lineType,
-                    env.entries,
-                    localLines,
-                    context,
-                    this.getTextureCoordinateType(technique) ? uvs : undefined,
-                    hasUntiledLines ? offsets : undefined
-                );
+                if (hasIndividualLineSegments) {
+                    assert(
```

Comment:
This assert could be inside the `if(...)` block below... but it just detail.

Reply:
Stupid me... Fixed.